tools/perf/components/cloud_storage.py

```diff
@@ -33,7 +33,7 @@ def UploadFileToCloudStorage(folder: CloudFolder, path: str):
 
   s3_url = f's3://{_CLOUD_BUCKET}/{folder}/{sha1}'
   success, _ = perf_test_utils.GetProcessOutput(
-      ['aws', 's3', 'cp', path, s3_url, '--acl', 'bucket-owner-full-control'])
```

Comment:
BTW, it works with these extra switches

Reply:
yes, but this is by default for CI creds, so we do not need it here